backtrader/feeds/psqlfeed

```diff
@@ -0,0 +1,101 @@
+# !/usr/bin/env python
+# -*- coding: utf-8; py-indent-offset:4 -*-
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+from backtrader import DataBase, date2num
+import datetime as dt
+import psycopg2
+from urllib.parse import urlparse
```

Comment:
Suggest moving those import statements to _connect_db method in order to minimize the overall Backtrader import time in case the PslqDatabase is not used.